tools/ci/taskcluster-run.py

```diff
@@ -23,10 +23,7 @@ def get_browser_args(product, channel, artifact_path):
         return ["--install-browser", "--processes=12"]
     if product == "chrome" or product == "chromium":
         # Taskcluster machines do not have GPUs, so use software rendering via --enable-swiftshader.
-        args = ["--enable-swiftshader"]
-        if channel == "nightly":
-            args.extend(["--install-browser", "--install-webdriver"])
-        return args
+        return ["--enable-swiftshader", "--install-browser", "--install-webdriver"]
```

Comment:
Will passing `--install-browser` still work for stable/beta/dev runs, or will those now fail because wptrunner doesn't know how to install them? Can you trigger test runs to check?

Reply:
I triggered stable and dev runs with these changes and things are running as expected (and the checks can be seen in the CI checks)